### ISSUE TITLE
Feature Request: Generate JSON object from YAML files in Workflow

### ISSUE THREAD

**Issue by Korving-F** [LivingInSyn/RMML#27]
Consuming the entire project is typically easier when using JSON objects (e.g., Tools like Sentinel / Defender can make use of an [externaldata operator](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/externaldata-operator?pivots=azuredataexplorer)).

It would be nice to see the YAML files being converted into a consumable JSON object through your github workflows.

See for example [Sysmon](https://github.com/olafhartong/sysmon-modular/blob/master/.github/workflows/config-build.yml) and [Loldrivers](https://github.com/magicsword-io/LOLDrivers/blob/main/bin/gen-files.py). 
This last one ties also into #17 since it shows an example for the auto sigma-rule creation logic.

See below a quick snippet I used to generate it for myself.
```python
import yaml
import json
from pathlib import Path

# Read in all RMM-definition files
files = Path('./RMMs').rglob('*.yml')

# Initialize final JSON array 
final_json_object = []

# Go over each file and append the YAML object
for file in files:
    with open(file,'r') as yaml_in:
        yaml_object = yaml.safe_load(yaml_in)
        final_json_object.append(yaml_object)

# Write out the final JSON to file
with open('rmm.json','w') as f:
    f.write(json.dumps(final_json_object))
```
```bash
# requires dependency installation
$ pip install pyyaml
```

P.s. Awesome project and congratulations ^_^ 
![image](https://github.com/LivingInSyn/RMML/assets/17042203/da9abd23-c292-4a2f-a7d9-3840355ded2f)

**Comment by LivingInSyn** [comment]
Would you want to see each RMM as an individual `json` file or one file with all of the RMMs?

**Comment by Korving-F** [comment]
One file with all of them combined.

**Comment by LivingInSyn** [comment]
Fixed in v1.4.0

https://github.com/LivingInSyn/RMML/releases/tag/v1.4.0
